partest-suite/src/test/resources/scala/tools/partest/scalajs/2.13.10/run/t8601d.sem

```diff
@@ -0,0 +1 @@
+nullPointers
```

Comment:
Did you check that there are no other partests that pass with these semantics? Maybe worth doing in an additional commit.